doc/rados/operations/crush-map.rst

```diff
@@ -225,106 +223,106 @@ You can view the contents of the rules with:
 Device classes
 --------------
 
-Each device can optionally have a *class* assigned.  By
-default, OSDs automatically set their class at startup to
-`hdd`, `ssd`, or `nvme` based on the type of device they are backed
-by.
+Each device can optionally have a *class* assigned. By default, OSDs
+automatically set their class at startup to `hdd`, `ssd`, or `nvme` in
+accordance with the type of device they are backed by.
```

Comment:
If `nvme` is automatically set, that's new -- NVMe devices in my experience show up as `ssd`.  But let's leave this as-is for now.